src/api/models/user.js

```diff
@@ -66,6 +66,9 @@ export default (sequelize, DataTypes) => {
     User.hasMany(models.Highlight, {
```

Comment:
This file is not supposed to be in this PR.
I've rebased and tried to remove it but I couldn't